apps/docs/src/guide/wallets/connectors.md

```diff
@@ -319,6 +319,9 @@ It can be instantiated as follows:
 
 <<< @/../../docs-snippets/src/guide/wallets/connectors.test.ts#fuel-instantiation-1{ts:line-numbers}
 
+> [!NOTE] Note
+> We recommend initializing the Fuel class with the `init` method to avoid the any potential race conditions that may arise from the async nature of instantiating a connector.
```

Comment:
```suggestion
> We recommend initializing the Fuel class with the `init` method to avoid any potential race conditions that may arise from the async nature of instantiating a connector.
```